resources/jolokia-access.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
One other thought I just had: `jolokia-access.xml` is the name the Jolokia docs assume when giving examples for this file. So, we should probably leave it as-is as it results in a better Google search.